app/models/foreman_docker/docker.rb

```diff
@@ -5,6 +5,10 @@ class Docker < ::ComputeResource
     attr_accessible :email
 
     validates :url, :format => { :with => URI.regexp }
+    validates :email, :format => { :with => /\A(([\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~]+((\.\"[\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+(\.[\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+)*\")*\.[\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~]+)*)|(\"[\w !#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+(\.[\w !#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+)*\"))
+                                  @[a-z0-9]+((\.[a-z0-9]+)*|(\-[a-z0-9]+)*)*\z/ix },
+    :length => { :maximum => 60 },
```

Comment:
It doesn't really come from anywhere, it's copied from user.rb https://github.com/theforeman/foreman/blob/develop/app/models/user.rb#L75